backend/auth/views.py

```diff
@@ -0,0 +1,49 @@
+import json
+from time import time
+from bson.objectid import ObjectId
+
+from aiohttp import web
+from aiohttp_session import get_session
+from aiohttp_security import remember, forget, authorized_userid
+
+
+def redirect(request, router_name):
+    url = request.app.router[router_name].url()
+    raise web.HTTPFound(url)
+
+
+def set_session(session, username, request):
```

Comment:
a little docstring would be good in this case. I think that I understand what the method does (from reading the code, not from the method name). And I suspect that the choice of method name is not great. But I would prefer not to pass judgment before I see the author's intent :-) 